docs: document Export JSON flyout for dashboards (#5909) (#6034)
elastic/docs-content: explore-analyze/dashboards/sharing.md

## Export dashboards [export-dashboards]

You can export a dashboard from {{kib}} in two formats. Use the table to choose the one that fits your goal.

| Format | Use it to | What it includes |
|---|---|---|

> | NDJSON | Move dashboards between spaces or clusters, back them up, or share them in bulk. | The selected dashboards along with their related data views and visualizations. Supports exporting multiple dashboards at once. |

Author: ```suggestion
| NDJSON | Move dashboards between spaces or clusters, back them up, or share them in bulk. | The selected dashboards along with their related objects, such as data views and visualizations. Supports exporting multiple dashboards at once. |
```

> | {applies_to}`stack: preview 9.4` {applies_to}`serverless: preview` JSON | Manage a single dashboard as code, version-control it, recreate it through the dashboards API, or inspect its stored state. | One dashboard, with the panel types and properties that the dashboards API supports. Any unsupported panel types and properties are listed in the flyout and removed from the export. |

Author: ```suggestion
| {applies_to}`stack: preview 9.4` {applies_to}`serverless: preview` JSON | Manage a single dashboard as code, version-control it, recreate it through the dashboards API, or inspect its stored state. | One dashboard, with the panel types and properties that the dashboards API supports. Any panel types and properties not supported by the API are listed in the export flyout and removed from the export. |
```

> | [JSON](#export-dashboard-json) {applies_to}`stack: preview 9.4` {applies_to}`serverless: preview` | Manage a single dashboard as code, version-control it, recreate it through the dashboards API, or inspect its stored state. | One dashboard, with the panel types and properties that the dashboards API supports. Any panel types and properties not supported by the API are listed in the export flyout and removed from the export. |

Reviewer: The stored state of a dashboard (serialized saved object) is quite different from the Dashboard API state. Maybe we just call it `state` not `stored state`?

```suggestion
| [JSON](#export-dashboard-json) {applies_to}`stack: preview 9.4` {applies_to}`serverless: preview` | Manage a single dashboard as code, version-control it, recreate it through the dashboards API, or inspect its state. | One dashboard, with the panel types and properties that the dashboards API supports. Any panel types and properties not supported by the API are listed in the export flyout and removed from the export. |
```

Author: Done — applied the suggestion. Thanks for the precision on the terminology.

![Option to export a dashboard](/explore-analyze/images/kibana-dashboard-export-saved-object.png "")

To automate {{kib}}, you can export dashboards as NDJSON using the [Export saved objects API](https://www.elastic.co/docs/api/doc/kibana/group/endpoint-saved-objects). It is important to export dashboards with all necessary references.

## Export a dashboard as JSON [export-dashboard-json]

> 

Author: ```suggestion
```

```{applies_to}
stack: preview 9.4
serverless: preview
```

> Export the JSON source of a single dashboard in a format that the Kibana dashboards API can consume. Use this option when you want to inspect the stored state of a dashboard, save it to a file, or send it to the API to recreate the dashboard in another space or instance.

Author: ```suggestion
Export the JSON source of a dashboard in a format that the {{kib}} dashboards API can consume. Use this option when you want to inspect the stored state of a dashboard, save it to a file, or send it to the API to recreate the dashboard in another space or instance.
```

![Option to export a dashboard](/explore-analyze/images/kibana-dashboard-export-saved-object.png "")

To automate {{kib}}, you can export dashboards as NDJSON using the [Export saved objects API](https://www.elastic.co/docs/api/doc/kibana/group/endpoint-saved-objects). It is important to export dashboards with all necessary references.

### Export as dashboards API-compatible JSON [export-dashboard-json]
```{applies_to}
stack: preview 9.4
serverless: preview
```

> Export the JSON source of a dashboard in a format that the {{kib}} dashboards API can consume. Use this option when you want to inspect the stored state of a dashboard, save it to a file, or send it to the API to recreate the dashboard in another space or instance.

Reviewer: ```suggestion
Export the JSON source of a dashboard in a format that the {{kib}} dashboards API can consume. Use this option when you want to inspect the state of a dashboard, save it to a file, or send it to the API to recreate the dashboard in another space or instance.
```

Author: Applied. Same change in this paragraph.
